plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -198,24 +198,19 @@ func findNodesThatFit(
 		}
 	}
 
-	if len(filtered) > 0 && len(extenders) != 0 {
-		for _, extender := range extenders {
-			filteredList, failedMap, err := extender.Filter(pod, filtered, nodeNameToInfo)
-			if err != nil {
-				return []*v1.Node{}, FailedPredicateMap{}, err
-			}
+	if len(filtered) > 0 && extender != nil {
+		filteredList, failedMap, err := extender.Filter(pod, filtered, nodeNameToInfo)
+		if err != nil {
+			return []*v1.Node{}, FailedPredicateMap{}, err
```

Comment:
Not yours; but I'd like to skip it or log if `err != nil`. I'm OK to keep it as is in this PR.